2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Current SDK version is 0.17.1
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Configure stats pid to 1486184
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Loading settings from /home/hawk/.config/wandb/settings
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Loading settings from /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/Thesis/wandb/settings
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_init.py:_log_setup():520] Logging user logs to /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/Thesis/wandb/run-20250424_183956-4mv5w8du/logs/debug.log
2025-04-24 18:39:56,290 INFO    MainThread:1486184 [wandb_init.py:_log_setup():521] Logging internal logs to /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/Thesis/wandb/run-20250424_183956-4mv5w8du/logs/debug-internal.log
2025-04-24 18:39:56,291 INFO    MainThread:1486184 [wandb_init.py:_jupyter_setup():466] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fbe112fe920>
2025-04-24 18:39:56,291 INFO    MainThread:1486184 [wandb_init.py:init():560] calling init triggers
2025-04-24 18:39:56,291 INFO    MainThread:1486184 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {'SNR': [10], 'CNR': [15], 'NU': [1], 'n_targets': 8, 'rand_n_targets': True, 'beta_end': 0.02, 'beta_start': 0.0001, 'batch_size': 16, 'num_epochs': 400, 'noise_steps': 2000, 'num_workers': 4, 'dataset_size': 5000, 'time_emb_dim': 256, 'learning_rate': 0.0001, 'scheduler_type': 'linear'}
2025-04-24 18:39:56,291 INFO    MainThread:1486184 [wandb_init.py:init():610] starting backend
2025-04-24 18:39:56,291 INFO    MainThread:1486184 [wandb_init.py:init():614] setting up manager
2025-04-24 18:39:56,293 INFO    MainThread:1486184 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-24 18:39:56,295 INFO    MainThread:1486184 [wandb_init.py:init():622] backend started and connected
2025-04-24 18:39:56,304 INFO    MainThread:1486184 [wandb_run.py:_label_probe_notebook():1334] probe notebook
2025-04-24 18:39:56,305 INFO    MainThread:1486184 [wandb_run.py:_label_probe_notebook():1344] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-24 18:39:56,305 INFO    MainThread:1486184 [wandb_init.py:init():711] updated telemetry
2025-04-24 18:39:56,314 INFO    MainThread:1486184 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2025-04-24 18:39:56,818 INFO    MainThread:1486184 [wandb_run.py:_on_init():2402] communicating current version
2025-04-24 18:39:57,175 INFO    MainThread:1486184 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.19.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-04-24 18:39:57,175 INFO    MainThread:1486184 [wandb_init.py:init():795] starting run threads in backend
2025-04-24 18:39:57,320 INFO    MainThread:1486184 [wandb_run.py:_console_start():2380] atexit reg
2025-04-24 18:39:57,320 INFO    MainThread:1486184 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-04-24 18:39:57,320 INFO    MainThread:1486184 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-04-24 18:39:57,320 INFO    MainThread:1486184 [wandb_run.py:_redirect():2325] Redirects installed.
2025-04-24 18:39:57,321 INFO    MainThread:1486184 [wandb_init.py:init():838] run started, returning control to user process
2025-04-24 18:39:57,321 INFO    MainThread:1486184 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-04-24 18:39:57,321 INFO    MainThread:1486184 [wandb_init.py:_pause_backend():431] pausing backend
2025-04-24 18:39:57,326 INFO    MainThread:1486184 [wandb_init.py:_resume_backend():436] resuming backend
2025-04-24 18:40:24,363 INFO    MainThread:1486184 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-04-24 18:40:24,363 INFO    MainThread:1486184 [wandb_init.py:_pause_backend():431] pausing backend
2025-04-24 18:40:24,392 INFO    MainThread:1486184 [wandb_init.py:_resume_backend():436] resuming backend
2025-04-24 18:40:24,393 INFO    MainThread:1486184 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-04-24 18:40:24,393 INFO    MainThread:1486184 [wandb_init.py:_pause_backend():431] pausing backend
2025-04-24 18:40:24,398 INFO    MainThread:1486184 [wandb_init.py:_resume_backend():436] resuming backend
2025-04-24 18:40:24,415 INFO    MainThread:1486184 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-04-24 18:40:24,415 INFO    MainThread:1486184 [wandb_init.py:_pause_backend():431] pausing backend
2025-04-24 18:40:24,422 INFO    MainThread:1486184 [wandb_init.py:_resume_backend():436] resuming backend
2025-04-24 18:40:24,430 INFO    MainThread:1486184 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-04-24 18:40:24,430 INFO    MainThread:1486184 [wandb_init.py:_pause_backend():431] pausing backend
2025-04-24 18:40:24,435 INFO    MainThread:1486184 [wandb_init.py:_resume_backend():436] resuming backend
2025-04-24 18:40:42,415 INFO    MainThread:1486184 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-04-24 18:40:42,415 INFO    MainThread:1486184 [wandb_init.py:_pause_backend():431] pausing backend
